src/arrayMethodReduce.js

```diff
@@ -5,7 +5,20 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let callbackResult = initialValue;
+    let i = 0;
```

Comment:
accumulator maybe will be better solution than callbackResult for name of variable and I think you can name variable i only in for loop in other situation would better name index

Reply:
Fix mate's comment